generators/app/templates/src/FOOBAR/package.json

```diff
@@ -20,46 +20,46 @@
     "@acpaas-ui/ngx-layout": "^4.2.1",
     "@acpaas-ui/ngx-pagination": "^4.2.1",
     "@acpaas-ui/ngx-table": "^4.5.1",
-    "@angular/animations": "10.1.6",
-    "@angular/cdk": "10.2.5",
-    "@angular/common": "11.0.0-next.4",
-    "@angular/compiler": "10.1.6",
-    "@angular/core": "10.1.6",
-    "@angular/forms": "10.1.6",
-    "@angular/platform-browser": "10.1.6",
-    "@angular/platform-browser-dynamic": "10.1.6",
-    "@angular/platform-server": "10.1.6",
-    "@angular/router": "10.1.6",
-    "@angular/service-worker": "10.1.6",
+    "@angular/animations": "13.3.7",
+    "@angular/cdk": "13.3.7",
+    "@angular/common": "13.3.7",
+    "@angular/compiler": "13.3.7",
+    "@angular/core": "13.3.7",
+    "@angular/forms": "13.3.7",
+    "@angular/platform-browser": "13.3.7",
+    "@angular/platform-browser-dynamic": "13.3.7",
+    "@angular/platform-server": "13.3.7",
+    "@angular/router": "13.3.7",
+    "@angular/service-worker": "13.3.7",
     "core-js": "3.6.4",
     "ngx-custom-validators": "8.0.0",
     "rxjs": "6.6.3",
     "tslib": "^2.0.0",
-    "zone.js": "~0.10.2"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
     "@a-ui/core": "^4.2.2",
```

Comment:
Is er een reden waarom acpaas/a-ui v5 niet gebruikt wordt?

Reply:
Ik heb dezelfde vraag ;-)

Reply:
Dit was niet echt mee opgenomen in de scope van de upgrade, maar ik zal zien hoever ik hiermee geraak.

Reply:
In principe zou dit altijd moeten meegenomen worden, aangezien dat bij collegebesluit is vastgelegd. Een andere goede reden om de upgrade mee te nemen, is toegankelijkheid. Versie 5 van de core branding is volledig geoptimaliseerd om WCAG 2.1 AA compliant te zijn. Alvast bedankt om er even naar te kijken en laat gerust iets weten als je problemen tegenkomt.

Reply:
@TriangleJuice, @goffincedric,
Aanpassingen doorgevoerd. Ik ben vertrokken van de https://github.com/digipolisantwerp/starter-kit-ui_app_nodejs repo om dit zo gelijk mogelijk te trekken.